package.json

```diff
@@ -6,9 +6,12 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
+    "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
+    "read-more-react": "^1.0.10",
```

Comment:
What is the `read-more-react` package doing?

Reply:
It's leftover haha - I thought it would do what I wanted it to, but it doesnt :| I decided I wanted the "Read More" to eventually link to the SingleReview Page, this does not assist me with that - I thought i uninstalled it, but I must've forgotten to delete it out of my dependencies!